src/auth.rs

```diff
@@ -0,0 +1,71 @@
+use std::io;
+
+use anyhow::{bail, format_err, Context};
+use serde::{Deserialize, Serialize};
+use toml_edit::Document;
+
+use crate::config::write_if_not_exists;
+use crate::home::Home;
+
+pub static MANIFEST_FILE_NAME: &str = "auth.toml";
+
+static DEFAULT_MANIFEST: &str = r#"
```

Comment:
same here

Reply:
No need. Want to keep consistency with the rest of the project that uses this.